ecs-agent/daemonimages/csidriver/update-gomock

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -euo pipefail
+
+mockgen -package driver -destination=./driver/mock_mount.go -source ./driver/mount.go
+
+# Fixes "Mounter Type cannot implement 'Mounter' as it has a non-exported method and is defined in a different package"
+# See https://github.com/kubernetes/mount-utils/commit/a20fcfb15a701977d086330b47b7efad51eb608e for context.
+sed -i '/type MockMounter struct {/a \\tmount_utils.Interface' ./driver/mock_mount.go
```

Comment:
I don't understand what issue is fixed by this. The link is also not helpful. Can you please provide more details on this thread?

Reply:
Without this, it won't add this `mount_utils.Interface` as one of the members of this mocked struct.

Reply:
Comment is copied from the EBS csi driver. And agree it's not helpful to understand. I can remove this.

Reply:
In the mount-util of version 0.27.1, which is the one we are using, the method mentioned in the commit is exported 
```
	// CanSafelySkipMountPointCheck indicates whether this mounter returns errors on
	// operations for targets that are not mount points. If this returns true, no such
	// errors will be returned.
	CanSafelySkipMountPointCheck() bool
```

Reply:
That's odd. Why does the mock need to implement `mount_utils.Interface`? If we need an interface that implements both `mount_utils.Interface` and `Mounter` then we should write a new interface that composes these two interfaces. This hack is a code smell to me.

Reply:
We can remove this sed, since currently Mounter interface doesn't has `mount_utils.Interface`
```
type Mounter interface {
	PathExists(path string) (bool, error)
}
```